src/main/java/school/lemon/changerequest/java/introduction/hw2/Task7.java

```diff
@@ -2,6 +2,13 @@
 
 public class Task7 {
     public static int countEvens(int[] arr) {
-        return 0;
+        int evenNum = 0;
+        for (int anArr : arr) {
+            if (anArr % 2 == 0) {
```

Comment:
There is faster way to check whether number is even or odd. Read about parity bit.